announcements/2020-Week-31/kvm.md

```diff
@@ -0,0 +1,11 @@
+# :zap: This Week in KVM Releases (July 31, 2020):zap:
+
+[KVM v12.1.0](https://github.com/giantswarm/releases/blob/master/kvm/v12.1.0) supports multiple NGINX Ingress Controllers per tenant cluster, moves the management of NGINX IC NodePort Service from kvm-operator to NGINX IC app itself, and fixes Quay being a single point of failure.
+
+**Optional Apps Releases**  
+(Can be installed from App Catalogs)
+
+1. 
```

Comment:
```suggestion
1. [NGINX IC v1.8.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#181---2020-07-28) made node ports configurable for NodePort Service type. (Note: NGINX is Optional and not Pre-Installed in AWS 10+ clusters)
2. [NGINX IC v1.8.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#180---2020-07-24) supported multiple NGINX IC App installations per tenant cluster, and dropped deprecated configuration properties. (Note: NGINX is Optional and not Pre-Installed in AWS 10+ clusters)
```

Reply:
@sslavic optional NGINX should not be in KVM yet, right?

Reply:
@sslavic thanks for adding to AWS!